### ISSUE TITLE
Japanese date format contains 月 (month) twice when using intl polyfill service

### ISSUE THREAD

**Issue by yloubry** [andyearnshaw/Intl.js#139]
I came across an issue today while using react-intl which relies on Intl.DateTimeFormat for date formatting.

Here are the scripts that are loaded: 

https://cdn.polyfill.io/v1/polyfill.min.js?features=Intl.~locale.en,Intl.~locale.fr,Intl.~locale.ja,Intl.~locale.ru

<FormattedDate value={new Date()} day="numeric" month="long" year="numeric" />

The result in Safari (relying on polyfill)

<img width="345" alt="screen shot 2015-10-01 at 2 04 28 pm" src="https://cloud.githubusercontent.com/assets/13892147/10233772/73e23e5e-6845-11e5-9486-40b9dde930ba.png">

The result in Chrome Version 45.0.2454.101 (64-bit)

<img width="372" alt="screen shot 2015-10-01 at 2 04 57 pm" src="https://cloud.githubusercontent.com/assets/13892147/10233771/73dece2c-6845-11e5-8a06-17a25b1f0c3a.png">

10 月 is the month of October but why is the long version containing the word 月 again?
The short version also produces the same output.

Here is a screenshot taken from the javascript console in Safari

<img width="827" alt="screen shot 2015-10-01 at 2 09 35 pm" src="https://cloud.githubusercontent.com/assets/13892147/10233896/34ec0cf6-6846-11e5-8659-dc7abc487d5f.png">

**Comment by okuryu** [comment]
Yes, it's a known issue. See #69.

**Comment by okuryu** [comment]
I think this will be fixed in https://github.com/Financial-Times/polyfill-service/pull/698.

**Comment by eschiebel** [comment]
I am seeing the same behavior in 'en', but only in Safari, and it seems only when served from a linux server. dates and times format correctly when served from Windows.

**Comment by watilde** [comment]
@eschiebel Hello. Thanks for reporting it. The issue will be fixed by https://github.com/andyearnshaw/Intl.js/pull/177.
I did test the following code with master branch I built in my local and it worked:

```
console.log(
  new IntlPolyfill.DateTimeFormat(
    "en", {
      year: "numeric",
      month: "long",
      day: "numeric",
      weekday: "long"
    }).format(new Date())
)
```

Related issue: https://github.com/andyearnshaw/Intl.js/issues/176

**Comment by caridy** [comment]
fixed in intl@1.2.2
